plugins/block-shareable-procedures/src/blocks.ts

```diff
@@ -1202,6 +1202,14 @@ const procedureCallerOnChangeMixin = {
   onchange: function (event) {
     if (this.disposed || this.workspace.isFlyout) return;
     if (event.type === Blockly.Events.BLOCK_MOVE) this.updateArgsMap_(true);
+    if (
+      event.type === Blockly.Events.BLOCK_CHANGE &&
+      event.blockId === this.id &&
+      event.element === 'disabled' &&
+      event.group
```

Comment:
Gotcha! makes sense.

Unfortunately if I remove event.group thepatch won't work.  This is because if you disable a `proc def` it will trigger a `proc call` enable/disable call and I have no idea how to distinguish it with disabling it directly.  Adding `event.group` in the UI seem to work but it doesn't feel like the right fix so I will close this PR for now. =)